.circleci/config.yml

```diff
@@ -83,6 +93,7 @@ jobs:
       - run:
           name: Helm Charts Lint and Template Render
           command: |
+            helm dependency update ./helm/
```

Comment:
nit: this chart has no dependencies (and below)

Reply:
removed